ep/lh/ep-lh-0002.md

```diff
@@ -49,6 +50,7 @@ Each engine must have `meta.json` file with the following properties:
 | -           | :-:      | :-:      | -           |
 | id          | string   | -        | Engine id, case insensitive unique identifier, required only if the engine name is not unique |
 | name        | string   | +        | Case sensitive official engine name, may not be unique |
+| alias      | string[]  | -        | List of engine aliases.  |
```

Comment:
Please follow the common style and remove the dot in the end.
Q: It's completely unclear what is it and why we need it? Even from the practical point of view, could you please find at least 3 linters with widely used aliases and equal output formats?

Reply:
Linters: (cpd) pmd, (grunt-lesslint) lesslint, (spotbugs) findbug, (mdl) markdown

Usually it's the linters who changed the name, but another linters use `parrent` and save output format

Reply:
Then please improve the description and how this alias may/should be used.